Automattic/newspack-plugin
# Onboarding: Services step; Newsletters wizard

### All Submissions:






### Changes proposed in this Pull Request:



Adds the Services step of the onboarding process, as well as a Newsletters wizard. 

### How to test the changes in this Pull Request:

1. Switch `newspack-newsletters` to [`add/update-settings` branch](https://github.com/Automattic/newspack-newsletters/pull/407)
2. Visit the Setup wizard, Services step (`/wp-admin/admin.php?page=newspack-setup-wizard#/services`)
3. Observe four sections, which can be toggled
4. Edit the first section ("Reader Revenue"), observe the settings are reflected in the Reader Revenue Wizard. Edit it there, too, and observe the settings updated when visiting the Services step again
5. Same with the next section ("Newsletters"), which maps to the "Authoring" part of the Engagement wizard
6. Observe that the last two sections (ad related) are mutually exclusive
7. Edit the ad related sections, observe changes reflected in the Advertising wizard
8. Visit the Engagement wizard, change the settings of the Newsletters and observe them being reflected in the setting of the Newspack Newsletters plugin 
9. Deactivate the Newspack Newsletters plugin. Observe the Engagement wizard displays a plugin card instead of the settings interface

### Other information:
